plugin-server/src/cdp/cdp-api.ts

```diff
@@ -79,6 +83,7 @@ export class CdpApi {
         router.get('/api/projects/:team_id/hog_functions/:id/status', asyncHandler(this.getFunctionStatus()))
         router.patch('/api/projects/:team_id/hog_functions/:id/status', asyncHandler(this.patchFunctionStatus()))
         router.get('/api/hog_function_templates', this.getHogFunctionTemplates)
+        router.post('/public/webhooks/mailjet/', asyncHandler(this.postMailjetWebhook()))
```

Comment:
This should be different - again this has _nothing_ to do with our generic webhooks product. We should just have something explicit like `public/messaging/mailjet_webhook`